docs/configuration/radio/device.mdx

```diff
@@ -11,24 +11,34 @@ The device config options are: Role, Serial Output, and Debug Log. Device config
+| TAK            | Optimized for ATAK system communication, reduces routine broadcasts.                                                     | Integration with ATAK systems for communication in tactical or coordinated operations.                                                 |
+| REPEATER       | Dedicated device that focuses solely on extending mesh network coverage by relaying messages. Not visible in Nodes list. | Best positioned in strategic locations to maximize the network's overall coverage not shown in topology.                               |
+| ROUTER         | Dedicated device that focuses solely on extending mesh network coverage by relaying messages. Visible in Nodes list.     | Best positioned in strategic locations to maximize the network's overall coverage shown in topology.                                   |
+| ROUTER_CLIENT  | Combination of both ROUTER and CLIENT.                                                                                   | Devices in a strategic position for priority routing that need to also serve as a standard CLIENT.                                     |                            
+
+#### Role Comparison
+| Device Role    | Bluetooth/Wi-Fi Enabled | Screen Enabled | Power Consumption | Retransmit | Prioritized Routing | Visible in Nodes List |
+| -------------- | ----------------------- | -------------- | ----------------- | ---------- | ------------------- | --------------------- |
+| CLIENT         | Yes                     | Yes            | Regular           | Yes        | No                  | Yes                   |
+| CLIENT_MUTE    | Yes                     | Yes            | Lowest            | No         | No                  | Yes                   |
+| CLIENT_HIDDEN  | Yes                     | Yes            | Lowest            | Known only | No                  | Yes                   |
+| TRACKER        | Optional                | No             | Low               | No         | No                  | Yes                   |
+| LOST_AND_FOUND | Optional                | No             | Regular           | No         | No                  | Yes                   |
+| SENSOR         | Optional                | No             | Lowest            | No         | No                  | Yes                   |
+| TAK            | Yes                     | Optional       | Low               | Yes        | No                  | Yes                   |
+| ROUTER         | No                      | No             | High              | Yes        | Yes                 | Yes                   |
+| ROUTER_CLIENT  | Yes                     | Yes            | Highest           | Yes        | Yes                 | Yes                   |
+| REPEATER       | No                      | No             | High              | Yes        | Yes                 | No                    |
```

Comment:
REPEATER has prioritized routing

Reply:
That's what this shows. Spreadsheet might be easier to read https://docs.google.com/spreadsheets/d/1sN0EzJEhlG_shaCdwolba_jQLpuApypmQ3KpjguMYpU/edit#gid=507821102